SwiftEvolve/Sources/SwiftEvolve/SyntaxExtensions.swift

```diff
@@ -19,8 +19,7 @@ import SwiftSyntax
 import Foundation
 
```

Comment:
These seem like they could be useful to have in SwiftSyntax itself. Not a blocker, just a thought.

Reply:
Yeah, that’s true. More traits seem useful in general.